File: CoreJava/interface-crud-app/src/components/CustomerDB.java
package components;

public class CustomerDB implements ICrudabe {

	@Override
	public void create() {
		System.out.println("Customer created.");

	}

	@Override
	public void read() {
		System.out.println("Customer Read.");

	}

	@Override
	public void update() {
		System.out.println("Customer Update.");

	}

	@Override
	public void delete() {
		System.out.println("Customer Delete.");

	}

}

File: C#/OOP/InterfaceCRUD-App/InterfaceCRUD-App/CustomerDB.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace InterfaceCRUD_App
{
    class CustomerDB : Icrudable
    {
     
	public void create() {
		Console.WriteLine("Customer created.");

	}

	public void read() {
        Console.WriteLine("Customer Read.");

	}

	public void update() {
        Console.WriteLine("Customer Update.");

	}

	public void delete() {
        Console.WriteLine("Customer Delete.");

	}
    }
}
